modules/node_modules/@frogpond/ccc-lib/calendar/google.mjs

```diff
@@ -1,17 +1,20 @@
 import {get} from '../http'
 import moment from 'moment'
+import _jsdom from 'jsdom'
+const {JSDOM} = _jsdom
 
 function convertGoogleEvents(data, now = moment()) {
 	let events = data.map(event => {
 		const startTime = moment(event.start.date || event.start.dateTime)
 		const endTime = moment(event.end.date || event.end.dateTime)
+		const description = JSDOM.fragment(event.description).textContent.trim()
```

Comment:
Let's do the simple thing first, and just change this to

```js
let description = event.description.replace('<br>', '\n')
description = JSDOM.fragment(description).textContent.trim()
```

so that we at least catch the `<br>` tags.